src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -58,7 +58,7 @@ class SideNav extends Component {
     }
 
     // Track non-nested sidenav item click: https://github.com/department-of-veterans-affairs/va.gov-team/issues/7643
-    if (depth === 2) {
+    if (depth === 1 || depth === 2) {
```

Comment:
#### depth 2 is this:
![image](https://user-images.githubusercontent.com/12773166/81229135-92e74b80-8fac-11ea-89ee-8fba58197553.png)

#### depth 1 is this:
![image](https://user-images.githubusercontent.com/12773166/81229209-aabecf80-8fac-11ea-98d3-eb0a096c6949.png)

Is this really want you want to do? 🙂

Reply:
Yeah, that won’t work, lol. @kelsonic.....I’m confused why parent level items aren’t registering?